config/locales/zh_TW.yml

```diff
@@ -0,0 +1,40 @@
+zh_TW:
+  errors:
+    messages:
+      taken_in_past: '曾被使用過。'
+      equal_to_current_password: '必須與目前密碼不同。'
+      password_complexity:
+        digit:
+          one: 必須包含至少一個數字
+          other: 必須包含至少 %{count} 個數字
+        lower:
+          one: 必須包含至少一個小寫字母
+          other: 必須包含至少 %{count} 個小寫字母
+        symbol:
+          one: 必須包含至少一個特殊符號
+          other: 必須包含至少 %{count} 個特殊符號
+        upper:
+          one: 必須包含至少一個大寫字母
+          other: 必須包含至少 %{count} 個大寫字母
+  devise:
+    invalid_captcha: '輸入的驗證碼無效。'
```

Comment:
<img width="1281" alt="Screen Shot 2020-10-20 at 4 05 03 PM" src="https://user-images.githubusercontent.com/1635820/96638385-05937b80-12ee-11eb-9120-e22a736b5d23.png">

Not sure if this is right? We aren't entering a verification code here.

Reply:
Verification code & recaptcha are actually translated into same Traditional Chinese word.

<img width="1078" alt="CleanShot 2020-10-21 at 12 28 08@2x" src="https://user-images.githubusercontent.com/14349/96673346-e69df380-1398-11eb-87d1-b222c2d31e81.png">

You can also find [Wikipedia page](https://zh.wikipedia.org/zh-tw/%E9%AA%8C%E8%AF%81%E7%A0%81) using this translation.